core/pfe-styles/demo/colors.html

```diff
@@ -125,15 +124,15 @@ <h2 id="colors">Color Palette</h2>
         <section class="pfe-l-grid pfe-m-gutters context-container">
           <p>Click the buttons below to change the theme context of the container:</p>
           <div class="context-buttons">
-            <pfe-cta priority="secondary" class="themes light">
+            <pfe-btton variant="secondary" class="themes light">
               <button name="light">Light theme</button>
-            </pfe-cta>
-            <pfe-cta priority="primary" color-palette="complement" class="themes dark">
+            </pfe-btton>
+            <pfe-btton variant="primary" color-palette="complement" class="themes dark">
               <button name="dark">Dark theme</button>
-            </pfe-cta>
-            <pfe-cta priority="primary" color-palette="base" class="themes saturated">
+            </pfe-btton>
+            <pfe-btton variant="primary" color-palette="base" class="themes saturated">
               <button name="saturated">Saturated theme</button>
-            </pfe-cta>
+            </pfe-btton>
```

Comment:
Typo fix

I'm also not sure this HTML page is displayed anywhere.  I couldn't find anything that linked to https://patternflyelements.org/core/styles/demo/colors.  Maybe able to delete it moving forward? 

```Suggestion
            <pfe-button variant="secondary" class="themes light">
              <button name="light">Light theme</button>
            </pfe-button>
            <pfe-button variant="primary" color-palette="complement" class="themes dark">
              <button name="dark">Dark theme</button>
            </pfe-button>
            <pfe-button variant="primary" color-palette="base" class="themes saturated">
              <button name="saturated">Saturated theme</button>
            </pfe-button>
```

Reply:
I'm happy deleting the pfe-styles package altogether, i'm just not sure who's using it.